recipe-client-addon/lib/FilterExpressions.jsm

```diff
@@ -41,3 +43,23 @@ this.FilterExpressions = {
     return jexl.eval(onelineExpr, context);
   },
 };
+
+/**
+ * Return an array of the given object's own keys (specifically, its enumerable
+ * properties).
+ * @param {Object} obj
+ * @return {Array[String]}
+ */
+function keys(obj) {
+  return Object.keys(obj);
+}
+
+/**
+ * Find all the values that are present in both lists.
+ * @param {Array} listA
+ * @param {Array} listB
+ * @return {Array}
+ */
+function operatorIntersect(listA, listB) {
+  return listA.filter(item => listB.includes(item));
```

Comment:
Yeah, `this` in JS is... interesting.

As an alternative, you could also pass `list2.includes.bind(list2)` which avoids explicitly declaring another function (but doesn't otherwise make much of a difference). <insert shruggy emoticon here>